telemetry/telemetry_record.py

```diff
@@ -29,11 +84,13 @@ def record(page_set, url, options='--browser=android-jb-system-chrome'):
     :param options: chromium browser options
     """
     record_path = os.path.join(CHROMIUM_SRC, 'tools/perf/record_wpr')
-    cmd = ' '.join([record_path, options, page_set])
+    cmd = ' '.join(['sudo', record_path, options, page_set])
+    print cmd
```

Comment:
Added a bit more printing here in case something failed. Ideally this should all go to a log file.